Libraries/Network/XMLHttpRequestBase.js

```diff
@@ -139,6 +143,15 @@ class XMLHttpRequestBase {
     }
   }
 
+  _stringToArrayBuffer(str: string): ArrayBuffer {
+    var buffer = new ArrayBuffer(str.length);
+    var bufferView = new Uint8Array(buffer);
```

Comment:
I only reverted the ArrayBuffer support. I've kept the `reponse` and `responseType` props (although they're kinda useless at the moment, since `text` is the only supported responseType value).